presentation/src/main/java/com/threegap/bitnagil/presentation/util/playstore/PlayStoreUtils.kt

```
@@ -32,45 +30,46 @@ fun openAppInPlayStore(
     } catch (e: ActivityNotFoundException) {
         false
     }
 
-private fun tryOpenWebBrowser(activity: ComponentActivity): Boolean =
+private fun tryOpenWebBrowser(activity: ComponentActivity, packageName: String): Boolean =
     try {
-        val intent = createWebIntent()
+        val intent = createWebIntent(packageName)
         activity.startActivity(intent)
         true
     } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)